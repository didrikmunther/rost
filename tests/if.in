fn main() {
	let a = 500

	if(a < 1000) {
		printf("a < 1000\n")
	} else {
		printf("a >= 1000\n")
	}

	if(a < 500) {
		printf("a < 500\n")
	} else {
		printf("a >= 500\n")
	}

	printf("Outside\n")
}