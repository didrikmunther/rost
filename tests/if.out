a < 1000
a >= 500
Outside